test/neural_net/CMakeLists.txt

```diff
@@ -6,6 +6,22 @@ if(APPLE AND
    NOT TC_BUILD_IOS AND 
    NOT TC_BASE_SDK_VERSION VERSION_LESS 10.15)
 
-  make_boost_test(style_transfer.cxx)
+  make_library(style_transfer_test_utils
+    SOURCES
+      style_transfer_testing.mm
+      style_transfer_testing_utils.mm
+    REQUIRES
+      unity_shared_for_testing
+  )
+
+  target_compile_options(style_transfer_test_utils 
+    PUBLIC 
+      "-fobjc-arc -Wno-nullability-completeness -Wno-shadow-ivar"
+  )
+  
+  make_boost_test(style_transfer.cxx
```

Comment:
Nit: for clarity, could we name this file `style_transfer_tests.cxx` or `test_style_transfer.cxx`?

Reply:
Sounds Good!